packages/features/ee/teams/pages/team-members-view.tsx

```diff
@@ -1,20 +1,20 @@
 import { MembershipRole } from "@prisma/client";
 import { useSession } from "next-auth/react";
 import { useRouter } from "next/router";
-import { useState } from "react";
+import { useState, Fragment } from "react";
```

Comment:
@zomars I noticed on other PRs you've used <Fragment> over <>. Is that the convention we should be aiming for?

Reply:
There's no convention. But I use `Fragment` when need to pass the `key` prop for arrays.